### ISSUE TITLE
Primer can break unexpectedly on main because we use bleeding edge of repositories

### ISSUE THREAD

**Issue by DanielNoord** [PyCQA/pylint#8998]
### Bug description

See https://github.com/nedbat/coveragepy/commits/master/pylintrc

Three weeks ago `coverage` moved their `pylintrc` which means the Primer fails on `main`.

I don't have access to a laptop with a checked out repo of `pylint` locally but we should probably update the `primer` configuration.

### Configuration

```ini
/
```

### Command used

```shell
/
```

### Pylint output

```shell
/
```

### Expected behavior

/

### Pylint version

```shell
/
```

**Comment by Pierre-Sassoulas** [comment]
I'm going to fix that. I wondered why it's failing on main and not on branches ?

**Comment by DanielNoord** [comment]
> I'm going to fix that. I wondered why it's failing on main and not on branches ?

Branches use the old cache and thus a commit where this doesn't occur yet.

**Comment by Pierre-Sassoulas** [comment]
Ha right thank you ! The long term solution seems to be what we discussed in https://github.com/pylint-dev/pylint/pull/8861#issuecomment-1665482114 then

**Comment by Pierre-Sassoulas** [comment]
See https://github.com/nedbat/coveragepy/commit/f2aa871b575aee0b10b2ec8f407b3963ec06c793, pylintrc moved to pyproject.toml. I'm going to open a PR shortly.
